braindecode/models/modules.py

```diff
@@ -356,3 +358,82 @@ def forward(self, x):
         return F.conv2d(
             x, weight=combined_weight, bias=bias, stride=(1, 1)
         )
+
+
+class MLP(nn.Module):
```

Comment:
I would use an MLP like this one https://github.com/probabilists/zuko/blob/master/zuko/nn.py#L122
here it's just one hidden layer.

Reply:
Thank you for this suggestion. I liked the Zuko library!